specification/eventgrid/data-plane/Microsoft.ServiceBus/stable/2020-12-03/ServiceBus.json

```diff
@@ -0,0 +1,131 @@
+{
+  "swagger": "3.0",
+  "info": {
+    "version": "2020-12-03",
+    "title": "Schema of Azure ServiceBus Messaging events published to Azure Event Grid",
+    "description": "Describes the schema of the Azure ServiceBus Messaging events published to Azure Event Grid. This corresponds to the Data property of an EventGridEvent."
+  },
+  "paths": {},
+  "definitions": {
+    "ServiceBusActiveMessagesAvailableWithNoListenersEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.ServiceBus.ActiveMessagesAvailableWithNoListeners event.",
+      "type": "object",
+      "properties": {
+        "namespaceName": {
+          "description": "The namespace name of the Microsoft.ServiceBus resource.",
+          "type": "string"
+        },
+        "requestUri": {
+          "description": "The endpoint of the Microsoft.ServiceBus resource.",
+          "type": "string"
+        },
+        "entityType": {
+          "description": "The entity type of the Microsoft.ServiceBus resource. Could be one of 'queue' or 'subscriber'.",
+          "type": "string"
+        },
+        "queueName": {
+          "description": "The name of the Microsoft.ServiceBus queue. If the entity type is of type 'subscriber', then this value will be null.",
+          "type": "string"
+        },
+        "topicName": {
+          "description": "The name of the Microsoft.ServiceBus topic. If the entity type is of type 'queue', then this value will be null.",
+          "type": "string"
+        },
+        "subscriptionName": {
+          "description": "The name of the Microsoft.ServiceBus topic's subscription. If the entity type is of type 'queue', then this value will be null.",
+          "type": "string"
+        }
+      }
+    },
+    "ServiceBusDeadletterMessagesAvailableWithNoListenersEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for a Microsoft.ServiceBus.DeadletterMessagesAvailableWithNoListenersEvent event.",
```

Comment:
In the previous version, this event has `Event` at the end. eg ActiveMessagesAvailableWithNoListeners ,DeadletterMessagesAvailableWithNoListenersEvent. But on the portal it is only ActiveMessagesAvailableWithNoListeners  and DeadletterMessagesAvailableWithNoListeners